README.rst

```diff
@@ -39,13 +39,29 @@ You can find our sources and single-click downloads:
 .. _Documentation: http://dipy.org
 .. _current trunk: http://github.com/nipy/dipy/archives/master
 
+
+Installing DIPY
+===============
+
+DIPY can be installed using `pip`:
+
+    pip install dipy
+
+or using `conda` (only on Python 2):
```

Comment:
Hello. As far as I know there are no issues with Python 3. Please remove.

Reply:
Did something change since our conversation two weeks ago?

Reply:
@arokem : The issue was OS X, not Python 3.  The current availability of VTK 7.1.1 is Python 2.7, 3.5 and 3.6 for linux-64, win-32 and win-64.  There are no `conda-forge` VTK packages for OS X (or for 32-bit linux).

I did recently upload VTK 7.1.1 packages built for OS X [in my channel](https://anaconda.org/grlee77/vtk) if you want to try them out.  As of now these match the `conda-forge` recipe, but these have not undergone the automated CI testing process and I can't guarantee to always keep them in sync going forward, so I wouldn't mention them in the README.